.gitignore

```diff
@@ -3,3 +3,4 @@ _site
 .jekyll-cache
 .jekyll-metadata
 vendor
+CONTRIBUTING.md
```

Comment:
Was this addition intentional?

Reply:
Yes. I want to have a working version with the bundle in, etc. before adding anything else; I didn't want to include this file. Probably should have stashed. I'll remove when I add the CONTRIBUTING.md file.